locales/bn/bn.json

```diff
@@ -6,16 +6,16 @@
     ":days day trial": ":days দিনের ট্রায়াল",
     ":resource Details": ":resource বিস্তারিত",
     ":resource Details: :title": ":resource বিবরণ: :title",
-    "A fresh verification link has been sent to your email address.": "A fresh verification link has been sent to your email address.",
-    "A new verification link has been sent to the email address you provided during registration.": "একটি নতুন যাচাইকরণ লিঙ্ক আপনার রেজিস্ট্রেশন করার সময় দেওয়া ইমেইল ঠিকানায় পাঠানো হয়েছে।",
+    "A fresh verification link has been sent to your email address.": "একটি নতুন ভেরিফিকেশন লিংক আপনার ই-মেইল ঠিকানায় প্রেরণ করা হয়েছে ।",
+    "A new verification link has been sent to the email address you provided during registration.": "একটি নতুন যাচাইকরণ লিঙ্ক আপনার রেজিস্ট্রেশন করার সময় দেওয়া ই-মেইল ঠিকানায় প্রেরণ করা হয়েছে ।",
     "Accept Invitation": "আমন্ত্রণ গ্রহণ",
-    "Action": "কাজ",
+    "Action": "কার্যকলাপ",
     "Action Happened At": "ঘটেছে",
     "Action Initiated By": "প্রকাশনার তারিখ",
     "Action Name": "নাম",
     "Action Status": "অবস্থা",
     "Action Target": "টার্গেট",
-    "Actions": "কাজ",
+    "Actions": "কার্যকলাপ",
     "Add": "যোগ করুন",
     "Add a new team member to your team, allowing them to collaborate with you.": "আপনার দলে একজন নতুন সদস্য যোগ করুন, তারা আপনার সাথে সহযোগিতা করতে সক্ষম হবেন।",
```

Comment:
```suggestion
    "Add a new team member to your team, allowing them to collaborate with you.": "আপনার দলে একজন নতুন সদস্য যোগ করুন, তারা আপনার সাথে সহযোগিতা করতে সক্ষম হবেন ।",
```

Reply:
The space before "dari" and no space both are correct in Bengali, I added a space for more readability. do you want I delete all space before "dari"?

Reply:
No, I want everything to be grammatically correct. I just see that you add spaces :)